openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TagRepository.java

```diff
@@ -250,6 +253,22 @@ public void entityRelationshipReindex(Tag original, Tag updated) {
           .getSearchClient()
           .reindexAcrossIndices("tags.tagFQN", original.getEntityReference());
     }
+
+    // Certification Updates
+    if (updated.getClassification().getFullyQualifiedName().equals("Certification")
```

Comment:
@mohityadav766 why are we only looking for Certification? shouldn't this be common for every classification?